lib/DxilPIXPasses/PixPassHelpers.cpp

```diff
@@ -203,10 +203,12 @@ std::vector<llvm::BasicBlock*> GetAllBlocks(hlsl::DxilModule& DM) {
     std::vector<llvm::BasicBlock*> ret;
     auto entryPoints = DM.GetExportedFunctions();
     for (auto& fn : entryPoints) {
-        auto & blocks = fn->getBasicBlockList();
-        for (auto & block : blocks) {
-            ret.push_back(&block);
+      if (fn != nullptr) {
```

Comment:
There are no null patch constant functions, this would be the null hull shader case (pass-through).  But see comment above about getting functions differently in the first place.